src/main/java/com/puppycrawl/tools/checkstyle/checks/FileContentsHolder.java

```diff
@@ -31,21 +31,20 @@
  * @author Mike McMahon
  * @author Rick Giles
  */
-public class FileContentsHolder
-    extends AbstractCheck {
+public class FileContentsHolder extends AbstractCheck {
     /** The current file contents. */
-    private static final ThreadLocal<FileContents> FILE_CONTENTS = new ThreadLocal<>();
+    private static FileContents currentFileContents;
 
     /**
      * @return the current file contents.
      */
-    public static FileContents getContents() {
-        return FILE_CONTENTS.get();
+    public static FileContents getCurrentFileContents() {
```

Comment:
@romani Isn't this breaking backwards compatibility renaming this `public static` method? Are we ok with this for a minor release?